Core/Versioning/ModuleVersion.cs

```diff
@@ -34,7 +34,7 @@ public partial class ModuleVersion
 
     public partial class ModuleVersion
     {
-        private readonly int _epoch;
+        private readonly ulong _epoch;
```

Comment:
We should also update the type of `Metadata.Epoch` in `Netkan/Model/Metadata.cs`, if `long`s are going to be supported here.